src/layout.rs

```diff
@@ -331,10 +331,7 @@ fn split(area: Rect, layout: &Layout) -> Rc<[Rect]> {
                     taffy
                         .new_leaf(Style {
                             // add node only with "width" set, because of layout.direction
-                            size: Size {
-                                width: dim(elt_applied as f32),
-                                height: auto(),
-                            },
+                            flex_basis: dim(elt_applied),
```

Comment:
What was the effect of this change rather than size?

Reply:
this is not in use anyone in the latest version of this PR, i had done this because it was smaller and equivalent to the previous version.

`flex-basis` is "size across the main axis", where the "main axis" is being defined via "flex_direction" (where `column` is height, and [`row (default)`](https://github.com/DioxusLabs/taffy/blob/dab541d6104d58e2e10ce90c4a1dad0b703160cd/src/style/mod.rs#L314) width), so it just made the code more clean instead of of having to have a `Size` struct with a `auto()` property

also see [this stackoverflow answer](https://stackoverflow.com/a/34355447/8944059)